mmv1/products/compute/FirewallPolicyWithRules.yaml

```diff
@@ -0,0 +1,450 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: FirewallPolicyWithRules
+min_version: beta
+base_url: 'locations/global/firewallPolicies?parentId={{parent}}'
+self_link: 'locations/global/firewallPolicies/{{policy_id}}'
+create_url: 'locations/global/firewallPolicies?parentId={{parent}}'
+update_verb: :PATCH
+description: "The Compute FirewallPolicy with rules resource"
```

Comment:
Can we improve this description? Something like:
FirewallPolicyWithRules declaratively manages all rules attached to a firewall policy

or something that describes why a user would want to use this over the regular FirewallPolicy

Reply:
done